### DISCUSSION TITLE
An option to disable server password?

### DISCUSSION THREAD

**Discussion by Verzacliche** [Pryaxis/TShock#2248] [Q&A]
I've already tried leaving a blank space when the the server asks for a password on startup, also tried editing the config.json but both still doesn't work, please help.

**Comment by Debug-pro-dev** [comment]
If you leave an empty space in the `ServerPassword` in your `config.json` and do not enter a password during initialization, then the server does not ask for it.

**Reply by Verzacliche** [reply]
I already tried that but when I initiated the server and left a blank space on the password it still asks for one when I join the server.
